src/components/ToolsPage/ToolsContainer.vue

```diff
@@ -0,0 +1,45 @@
+<template>
```

Comment:
Instead of this being in `src/components/ToolsPage/ToolsContainer.vue` I think you should just call it `Tools.vue` and put it under the `/containers` folder